st2common/st2common/logging/handlers.py

```diff
@@ -37,7 +38,8 @@ def __init__(self, filename, mode='a', maxBytes=0, backupCount=0, encoding=None,
         format_values = {
             'timestamp': timestamp,
             'ts': isotime_str,
-            'pid': pid
+            'pid': pid,
+            'pack': os.path.abspath(os.path.dirname(sys.argv[2])).split('/')[-2]
```

Comment:
Yes Kami your understanding is correct. we tried to get the pack name from the command line arguments.  As per your suggestion now We are looking into alternate way to get the pack name.